tools/setup/emoji/build_emoji

```diff
@@ -153,6 +155,9 @@ def code_point_to_file_name_map(ttx):
 
 def main():
     # type: () -> None
+    run(["sudo", "mkdir", "-p", EMOJI_CACHE_PATH])
+    run(["sudo", "chown", "%s:%s" % (user_id, user_id), EMOJI_CACHE_PATH])
```

Comment:
This also won't work in production -- we don't require `sudo` to be installed on Zulip production systems.